### ISSUE TITLE
Unable to use create glue in protected claim

### ISSUE THREAD

**Issue by PooPooSkin** [thexaero/open-parties-and-claims#513]
I'm not sure if I am just missing some config option, but I've disabled most, if not all, player/other/redirect protections for the server-claims-config, but I'm still unable to glue any blocks with Create glue within server claimed chunks. 

I mainly just would like the server claimed chunks to only protect from mob griefing, fire, and explosions.

**Comment by thexaero** [comment]
Yeah, I think protection for that is always enabled if protection is enabled at all. I'll probably make it check the "Blocks From Players" option.

**Comment by PooPooSkin** [comment]
Ok cool. I'll wait eagerly for an update if you fix it.

**Comment by wiloxe** [comment]
If you don't mind me asking, any updates on this?

**Comment by stacode123** [comment]
any updates?
